app/src/controllers/organizations/organization/reducer.js

```diff
@@ -16,17 +16,14 @@
 
 import { combineReducers } from 'redux';
 import { fetchReducer } from 'controllers/fetch';
-import { FETCH_ORGANIZATION_PROJECTS, SET_ACTIVE_ORGANIZATION } from './constants';
+import { loadingReducer } from 'controllers/loading';
+import { FETCH_ORGANIZATION_BY_SLUG } from './constants';
 
-export const activeOrganizationReducer = (state = null, { type = '', payload = {} }) => {
-  switch (type) {
-    case SET_ACTIVE_ORGANIZATION:
-      return payload;
-    default:
-      return state;
-  }
-};
 export const organizationReducer = combineReducers({
-  projects: fetchReducer(FETCH_ORGANIZATION_PROJECTS, { contentPath: 'content' }),
```

Comment:
Looks like you use the reducer defined in `controllers/administrate/projects`.
Please move them here together with other stuff as they are now related to the organizations and administrate part will be removed soon.